### ISSUE TITLE
Pytest Boolean Assertion Error

### ISSUE THREAD

**Issue by bpaul4** [IDAES/idaes-pse#692]
I noticed two new errors in GitHub Actions runs this morning in `idaes/dmf/tests/test_util.py::test_datetime_timestamp` and `idaes/generic_models/properties/core/eos/tests/test_ceos_PR.py::test_common`. Both errors gave the same output message:

```
E               AssertionError: approx() is not supported in a boolean context.
E               Did you mean: `assert a == approx(b)`?
```

from calling `pytest.approx()' methods. I traced this back to its source (https://github.com/pytest-dev/pytest/blob/57f8f5d2b313b9601872a1184a809060468a606e/src/_pytest/python_api.py#L103) and believe this may be a Pyomo or Python issue from a recent update. I did not see any IDAES or Pyomo commits this morning that may have caused this, and am curious what @lbianchi-lbl thinks of this.

**Comment by andrewlee94** [comment]
@bpaul4 I saw these as well, and they are due to a not that recent change in Pyomo (I am not sure why it took so long for these to show up). The issues is that one side of the assertion is a Pyomo expression which is not wrapped by the `value` function,

**Comment by bpaul4** [comment]
@andrewlee94 okay, I can open a PR to fix these two tests if that is the best way to go. Curiously, I did not see these failures locally although the Windows environments failed on GitHub Actions even after updating my local `main` branch.

**Comment by lbianchi-lbl** [comment]
@bpaul4 thanks for opening an issue about this. It looks like pytest just released a new major version (7.0.0) a few hours ago (https://pypi.org/project/pytest/#history), which I guess could be the reason why this error is only showing up now. The fix would be to either pin the pytest major version to 6, or, if these are the only parts of the codebase that are causing issues with pytest 7, try to address the issue to maintain compatibility without pinning.

**Comment by andrewlee94** [comment]
@lbianchi-lbl We have a PR open to fix the tests themselves - they are minor changes so we shouldn't be pinning `pytest` over something this small.
